src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,31 +128,34 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        //ensuring same mode of operation is used when the data is less than 1 block long
+        String plaintextLength28 = StringHelper.generateStringOfLength(7, 'A');
+        assertEncryptString(plaintextLength28);
+
+        //ensuring same mode of operation is used when the data is equal to 1 block
+        String plaintextLength128 = StringHelper.generateStringOfLength(32, 'A');
+        assertEncryptString(plaintextLength128);
+
+        //check if padding is applied, ensure same mode of operation is used when the data is more than 1 block long
+        String plaintextLength136 = plaintextLength128 + StringHelper.generateStringOfLength(2, 'A');
+        assertEncryptString(plaintextLength136);
+    }
+
     private static String encryptWithoutSpecifyingAlgorithmParams(String value) throws Exception {
 
         SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
         Cipher cipher = Cipher.getInstance("AES");
         cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
         byte[] encrypted = cipher.doFinal(value.getBytes());
         return StringHelper.byteArrayToHexString(encrypted);
-
     }
 
-    @Test
-    public void testDefaultAesCipherParams() throws Exception {
-        String plaintextLength28 = "AAAAAAA";
-        //Using plaintext > 128 bits to check any differences operations.
-        String plaintextLength128 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCCCC";
-        String plaintextLength136 = plaintextLength128 + "AA";
-        String expectedCiphertext;
-        String actualCiphertext;
-        String[] inputMsgs = {plaintextLength28, plaintextLength128, plaintextLength136};
-        for (String msg : inputMsgs) {
-
-            actualCiphertext = encryptWithoutSpecifyingAlgorithmParams(msg);
-            expectedCiphertext = StringHelper.encrypt(msg);
-            assertEquals(expectedCiphertext, actualCiphertext);
-        }
+    private static void assertEncryptString(String value) throws Exception {
```

Comment:
Is it good ?
        //verifying encryption string is same for default AlgorithmParams and with specified AlgorithmParams
        private static void assertEncryptString(String value) throws Exception {
        String actualCiphertext = encryptWithoutSpecifyingAlgorithmParams(value);
        String expectedCiphertext = StringHelper.encrypt(value);
        assertEquals(expectedCiphertext, actualCiphertext);
    }`